src/test/java/br/unb/cic/analysis/svfa/confluence/SVFAConfluenceAnalysisWithMultipleConfluences.java

```diff
@@ -0,0 +1,56 @@
+package br.unb.cic.analysis.svfa.confluence;
+
+import br.unb.cic.analysis.AbstractMergeConflictDefinition;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class SVFAConfluenceAnalysisWithMultipleConfluences {
```

Comment:
@rafaelmotaalves is this a test? shouldn't it be in another place?

Reply:
yes, it is, i was following the pattern I saw on the rest of the project adding the tests to the same package as the tested analysis but on the src/test folder, should it move it? @pauloborba

Reply:
Just saw it, it was on the wrong folder